src/efivar.c

```diff
@@ -53,34 +53,6 @@ static const char *attribute_names[] = {
 	""
 };
 
-static inline void
```

Comment:
You  need a commit message explaining why you are changing this

Reply:
Done